src/styles/card.scss

```diff
@@ -0,0 +1,76 @@
+.card {
+  border: 1px solid var(--gray-elements);
+  border-radius: 5px;
+  width: 198px;
+  height: 406px;
+  background-color: var(--white);
+  justify-content: center;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 36px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 150%;
+  color: #060b35;
+  margin: 0 16px 4px;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 140%;
+  color: #616070;
+  margin-top: 4px;
+}
+
+.card__price {
+  display: flex;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 150%;
+  color: var(--secondary);
+  margin-left: 16px;
+  margin-bottom: 16px;
+}
+
+.card__price-value {
+  display: block;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 112%;
+  text-align: right;
+  margin-left: 85px;
+  color: var(--main-accent);
+}
+
+.card__button-buy {
+  font-family: var(--font-family);
```

Comment:
Ensure that the SCSS variable `--font-family` is defined in a separate variables file and imported correctly.